Add readme file. (#1)
balena-io-playground/balenalib-systemd-example: README.md

# Balenalib Systemd Example

> On january 2019, [balena.io](https://balena.io) relesaed a set of 26K new base images ([read more](https://www.balena.io/blog/new-year-new-balena-base-images/)).

Reviewer: ```suggestion
In January 2019, [balena](https://balena.io) released a set of 26K new base images ([read more](https://www.balena.io/blog/new-year-new-balena-base-images/)).
```

> The new images does not contain `systemd` built-in, 

Reviewer: ```suggestion
The new images do not have `systemd` built-in. 
```

> This is an exemple code showing how to add it to a current or new project.

Reviewer: ```suggestion
This is an example showing how to add it to a new or existing project.
```
